Source/src/resources/ResourceMesh.h

```diff
@@ -6,23 +6,38 @@ namespace Hachiko
     class ResourceMesh : public Resource
     {
     public:
+        struct Bone
+        {
+            char name[PATH_MAX]; // TODO: find a better size
```

Comment:
If the size is good as is, let's delete this TODO.

Reply:
oh sorry yeah